core/mondoo-github-security.mql.yaml

```diff
@@ -416,7 +539,7 @@ queries:
       if ( github.organization.repositories.one(name == ".github") ) {
         github.organization.repositories.where( name == ".github") {
           files.one( name.downcase == "support.md")
-        }
+        } || github.repository.files.one( name.downcase == "support.md")
```

Comment:
a where and `||` check does not work well together

Reply:
@chris-rock The challenge is I uncovered a condition we need to handle where there is a `.github` folder with no `SUPPORT.md` file, but  at the same time the repo itself has a `SUPPORT.md`. As the test was originally written, the query would immediately fail the first test, and never check to see if the repo itself contained the required file. Any thoughts on how to address this case? 🤔

Reply:
I'll can update the query